english/deny/deny.md

```diff
@@ -0,0 +1,2 @@
+You shut your eyes and deny the existence of the marshmallows. You convince yourself they do not exist.
+Opening your eyes your field of vision is white. All white. Is this paradise? Is this fluffy sugar? You blink but the world remains white.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.